ldap/georchestra.ldif

```diff
@@ -221,12 +222,75 @@ member: uid=testuser,ou=users,dc=georchestra,dc=org
 o: Project Steering Committee
 ou: PSC
 seeAlso: o=psc,ou=orgs,dc=georchestra,dc=org
+seeAlso: uid=psc,ou=orgs,dc=georchestra,dc=org
 
+dn: uid=psc,ou=orgs,dc=georchestra,dc=org
+objectClass: top
+objectClass: person
+objectClass: organizationalPerson
+objectClass: inetOrgPerson
+cn: psc
+sn: psc
+uid: psc
+jpegPhoto:: /9j/4AAQSkZJRgABAQAAAQABAAD/2wBDAAgGBQgMFBofBgYHCg0dHhwHBwgMFB0j
```

Comment:
So you can add a `jpegPhoto` attribute onto a `inetOrgPerson` object, but it seems that the original request was on the organizations objects, right ? (either on the `organization` or the `groupOfMembers` object)

Reply:
There's no `jpegPhoto` attribute on `organizations` or the `groupOfMembers` objects.
The trick here is to reference a `inetOrgPerson` instance via the `seeAlso` attribute of the `groupOfMembers` instance

Reply:
So, one new object for managing the organization, that would have some consequences also on the console code :(

Reply:
True, this is planned.